py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -252,9 +264,16 @@ def _get_connection_manager(self):
 
         return urllib3.PoolManager(**pool_manager_init_args)
 
-    def __init__(self, remote_server_addr: str, keep_alive: bool = False, ignore_proxy: bool = False):
+    def __init__(
+        self,
+        remote_server_addr: str,
+        keep_alive: bool = False,
+        ignore_proxy: bool = False,
+        ignore_certificates: bool = False,
```

Comment:
Could you let users give `init_args_for_pool_manager`?
```
    def __init__(
        self,
        remote_server_addr: str,
        keep_alive: bool = False,
        ignore_proxy: bool = False,
        ignore_certificates: bool = False,
        init_args_for_pool_manager=None,
    ):
        self.keep_alive = keep_alive
        self._url = remote_server_addr
        self._ignore_certificates = ignore_certificates
        self._init_args_for_pool_manager = {} if init_args_for_pool_manager is None else init_args_for_pool_manager
```

Then, `_get_connection_manager` call in this file will be:
```
self._get_connection_manager(init_args_for_pool_manager=self._init_args_for_pool_manager)
```

or modify `_get_connection_manager` to:

```
    def _get_connection_manager(self):
        pool_manager_init_args = {"timeout": self.get_timeout()}
        pool_manager_init_args.update(self._init_args_for_pool_manager.get("init_args_for_pool_manager", {}))
```

Currently we need to override `_get_connection_manager` method call with arguments

Reply:
Ok, I will implement this